Platform/CoffeelakeBoardPkg/Script/StitchIfwiConfig.py

```diff
@@ -0,0 +1,202 @@
+## @ StitchIfwi.py
+#  This is an IFWI stitch config script for Slim Bootloader
+#
+# Copyright (c) 2019, Intel Corporation. All rights reserved. <BR>
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+extra_usage_txt = \

```

Comment:
Usage text should be here instead of StitchIfwi.py?

Reply:
okay I will move it back to stitchifwi